.gitmodules

```diff
@@ -4,12 +4,15 @@
 [submodule "ethexe/contracts/lib/openzeppelin-contracts"]
```

Comment:
plz also add tag for forge-std

Reply:
> Specify last release tag for openzeppelin libs

it's ok, 5.1.0 is latest version of openzeppelin: https://github.com/OpenZeppelin/openzeppelin-contracts/releases